core/utils/utils.go

```diff
@@ -17,6 +17,8 @@ import (
 	"sync/atomic"
 	"time"
 
+	stderrs "errors"
```

Comment:
It would be nice to have repo wide standards for these so we don't get confused, but out of scope here. I'm partial to leaving std lib as `errors` and using `pkgerrors` for pkg/errors though 🤷 would that be a noisy local change?

Reply:
I completely agree it's gotten me confused multiple times. Let me send another PR with that change across the repo